services/content-publishing/apps/api/src/main.ts

```diff
@@ -21,8 +33,10 @@ async function bootstrap() {
 
   // Get event emitter & register a shutdown listener
   const eventEmitter = app.get<EventEmitter2>(EventEmitter2);
+  eventEmitter.setMaxListeners(20);
```

Comment:
Do we also need to increment the max listeners in account-service?

Reply:
Probably not; `content-publishing` was logging warning & errors about leaking resources due to exceeding max listeners, it think mainly due to the number of queues and things. `account-service` isn't issuing that warning, so probably fine.